tests/phpunit/tests/test-affordabilitywidget.php

```diff
@@ -84,4 +84,79 @@ public function testgetCustomisation()
 
         $this->assertSame('#8BBFFF', $response);
     }
+
+    public function testisEnabled()
+    {
+        $this->assertSame('true', isEnabled('rzp_afd_enable_dark_logo'));
+
+        add_option('rzp_afd_enable_dark_logo', 'yes');
+
+        $this->assertSame('true', isEnabled('rzp_afd_enable_dark_logo'));
+    }
+
+    public function testgetFooterDarkLogo()
+    {
+        $this->assertSame('true', getFooterDarkLogo());
+    }
+
+    public function testgetPayLater()
```

Comment:
Write UT for paylater, cardless emi, emi when the limited_providers are not present.

Reply:
Added